hw/ip/aon_timer/doc/_index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "AON Timer Technical Specification"
+---
+
+# Overview
+
+This document specifies the Always-On ("AON") Timer IP functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification" >}})
+See that document for an overview of how it is integrated into the top level system.
+
+## Features
+
+- Two 32-bit upcounting timers: one timer functions as a wakeup timer, one as a watchdog timer
+- The watchdog timer has two thresholds: bark (generates an interrupt) and bite (resets core)
+
+## Description
+
+### AON Wakeup timer
+
+The always-on wakeup timer operation is straightforward.
+A count starts at 0 and slowly ticks upwards.
+When it reaches / exceeds the wake count, a level wakeup signal is sent to the power manager and a level IRQ is generated to the processor.
+This wakeup signal stays high until it is explicitly acknowledged by software.
+The wakeup timer can be used like a real-time clock for long periods of low-power mode (though it does not give any guarantees of time-accuracy TODO: specify accuracy).
+
+### AON Watchdog timer
+
+The always-on watchdog timer behaves similarly to the wakeup timer.
+It has an independent count starting at 0 which slowly ticks upwards.
+When the first threshold is met or exceeded, a level wakeup signal (if enabled) is generated to the power manager.
```

Comment:
Maybe "sent to the power manager"? (Or "generated and sent to", but I'm not sure that's needed for a wire going high!)

Reply:
Will fix